AP_Randomizer/apworld/rules_expert.py

```diff
@@ -6,11 +6,30 @@ def __init__(self, world) -> None:
         super().__init__(world)
 
         region_clauses = {
+            "Empty Bailey -> Upper Bailey": lambda state:
+                self.has_slide(state),
+            "Upper Bailey -> Tower Remains": lambda state:
+                self.has_slide(state)  # is a bit tight, consider adding a kick to make it easier?
+                or self.kick_or_plunge(state, 3),  # maybe doesn't need to be here
```

Comment:
just slide is fine imo, and you can get rid of kick_or_plunge 3 since that's in hard

Reply:
i was going back and forth on where i wanted the 4/3 split and didn't notice i had 3 in two placed after i settled on normal/hard lol
```suggestion
                self.has_slide(state)
```